CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## [v11.16.3](https://github.com/auth0/lock/tree/v11.16.3) (2019-06-11)
+[Full Changelog](https://github.com/auth0/lock/compare/v11.16.2...v11.16.3)
+
+
```

Comment:
Where's the changelog?

Reply:
ops. I had a typo in the milestone. fixed